plugin-server/package.json

```diff
@@ -75,6 +75,7 @@
         "ioredis": "^4.27.6",
         "ipaddr.js": "^2.1.0",
         "kafkajs": "^2.2.0",
+        "kafkajs-lz4": "2.0.0-beta.0",
```

Comment:
was going to complain about beta, but we've been running alpha rrweb for 12 months 🤣